php/class-coauthors-guest-authors.php

```diff
@@ -1011,6 +1011,13 @@ function get_guest_author_thumbnail( $guest_author, $size, $class = null ) {
 	 */
 	function get_guest_author_fields( $groups = 'all' ) {
 
+		if ( get_site_option( 'initial_db_version' ) < 23588 ) {
```

Comment:
What if the site is not a multisite?

Reply:
@rebeccahum The screenshots of this patch working are from a single-site installation. I'm not aware that `initial_db_version` works differently in a multisite:

https://developer.wordpress.org/reference/functions/wp_version_check/
https://github.com/WordPress/WordPress/blob/master/wp-includes/update.php#L95-L104

Please correct me if I'm wrong!